terraform-modules/k8s/k8s-cluster.tf

```diff
@@ -59,36 +55,37 @@ resource "google_container_cluster" "cluster" {
   # CIS compliance: Enable Alias IP Ranges. According to the terrafform
   # docs, setting these values blank gets default-size ranges automatically
   # chosen: https://www.terraform.io/docs/providers/google/r/container_cluster.html#ip_allocation_policy
-  ip_allocation_policy = ["${var.ip_allocation_policy}"]
+  ip_allocation_policy = var.ip_allocation_policy
 
   # CIS compliance: Enable PodSecurityPolicyController
   pod_security_policy_config {
     enabled = true
   }
 
-  # OMISSION: CIS compliance: Enable Private Cluster 
-  private_cluster_config = ["${var.private_cluster_config}"]
+  # OMISSION: CIS compliance: Enable Private Cluster
+  private_cluster_config {
+    enable_private_endpoint = var.enable_private_endpoint
+    enable_private_nodes = var.enable_private_nodes
+    master_ipv4_cidr_block = var.private_master_ipv4_cidr_block
+  }
```

Comment:
@RLuckom I asked @smark-broad if it'd make sense for there to be > 1 `private_cluster_config` and he didn't think so, so I inlined it here. Let me know if I should revert back to `dynamic` usage to support 0-N blocks.

Reply:
this looks right